### DISCUSSION TITLE
Can I render dynamic object in mitsuba3 ?

### DISCUSSION THREAD

**Discussion by erqihuang** [mitsuba-renderer/mitsuba3#1320] [Q&A] [answered]
## Summary
Can I render dynamic object in mitsuba3 ?

**Comment by njroussel** [comment, marked as answer]
Hi @erqihuang 

What do you mean by dynamic object?

If you're talking about moving shapes in the scene, and rendering them with motion blur, this is unfortunately not supported at this time.

**Reply by erqihuang** [reply]
Hi @njroussel

Thank you for your reply.

Yes, that is what I meant. I expect to experience this feature in the future.
